README.md

```diff
@@ -415,29 +403,27 @@ See [BIGTREETECH-TouchScreenFirmware/releases](https://github.com/bigtreetech/BI
 
 Octoprint can optionally trigger some actions to the TFT sending specific gcodes. The following actions and the related triggering gcodes are currently supported by the TFT fw:
 
-**start:**
-M118 A1 P0 action:print_start
-  
-**end:**
-M118 A1 P0 action:print_end
-  
-**cancel:**
-M118 A1 P0 action:cancel
-  
-**pause:**
-M118 A1 P0 action:pause
-  
-**resume:**
-M118 A1 P0 action:resume
-  
-**time remaining progress:**
-M118 A1 P0 action:notification Time Left hms (e.g. 02h04m06s)
-  
-M117 Time Left hms
-**file data progress:**
-  
-M118 A1 P0 action:notification Data Left / (e.g. 123/12345)
-M117 Data Left /
+**start:**<br>
+`M118 A1 P0 action:print_start`
+
+**end:**<br>
+`M118 A1 P0 action:print_end`
+
+**cancel:**<br>
+`M118 A1 P0 action:cancel`
+
+**pause:**<br>
+`M118 A1 P0 action:pause`
+
+**resume:**<br>
+`M118 A1 P0 action:resume`
+
+**time remaining progress:**<br>
```

Comment:
In the current readme file there were some missing info (due to characters requiring escape char "\\" first) in these last two commands.
The right syntax for the commands is:

**time remaining progress:**
M118 A1 P0 action:notification Time Left \<XX\>h\<YY\>m\<ZZ\>s (e.g. 02h04m06s)
or
M117 Time Left \<XX\>h\<YY\>m\<ZZ\>s

**file data progress:**
M118 A1 P0 action:notification Data Left \<XXXX\>/\<YYYY\> (e.g. 123/12345)
or
M117 Data Left \<XXXX\>/\<YYYY\>

Reply:
Oh. ok.

Reply:
thanks for the changes. There is just a missing \<br\> after M118 A1 P0 action:notification Data Left \<XXXX\>/\<YYYY\> (e.g. 123/12345). It is instead properly present for M118 A1 P0 action:notification Time Left \<XX\>h\<YY\>m\<ZZ\>s (e.g. 02h04m06s)

Reply:
ok